plugins/modules/apt_rpm.py

```diff
@@ -97,10 +129,33 @@ def query_package_provides(module, name):
 
 
 def update_package_db(module):
-    rc, out, err = module.run_command("%s update" % APT_PATH)
+    rc, out, err = module.run_command(["md5sum", "/var/cache/apt/pkgcache.bin"])
+    md5sum_before = out.split()[0]
+    rc, update_out, err = module.run_command([APT_PATH, "update"], check_rc=True, environ_update={"LANG": "C"})
+    rc, out, err = module.run_command(["md5sum", "/var/cache/apt/pkgcache.bin"])
+    md5sum_after = out.split()[0]
+    return (md5sum_before != md5sum_after, update_out)
+
+
+def dir_size(module, path):
+    rc, out, err = module.run_command(["du", "-sb", path], check_rc=True)
```

Comment:
Don't use external utils when you can do this easily with Python code. Using external programs adds new dependencies, which is technically a breaking change.